src/wallet/index.ts

```diff
@@ -72,6 +72,20 @@ class Wallet {
     return Wallet.deriveWallet(seed, algorithm)
   }
 
+  /**
+   * Derives a wallet from a secret (AKA a seed)
+   *
+   * @param secret - A string used to generate a keypair (publicKey/privateKey) to derive a wallet.
+   * @param algorithm - The digital signature algorithm to generate an address fro.
+   * @returns A Wallet derived from a secret (AKA a seed).
+   */
+  public static fromSecret(
```

Comment:
You might be able to do this more simply by just doing `public static fromSecret = fromSeed`

Reply:
Good to know. I think for this case it's good to have it like this so that the documentation gets picked up by docgen. My thought process is that `fromSecret` is not inferior to `fromSeed` from a naming perspective, and I don't think an alias using `public static fromSecret = fromSeed` would get picked up in docgen. (Although I could be wrong)

Reply:
there's got to be a way to document object attributes, IE `fromSecret = fromSeed`. if you just add the docstrings for this above that assignment I would expect it to work

Reply:
You're right, that works - Switched to @mvadari's simpler syntax :)